app/views/application/_location_form.html.erb

```diff
@@ -4,7 +4,12 @@
       data-track-action="postcodeSearchStarted">
 
   <% if @location_error %>
-    <div class="error-summary">
+    <div class="error-summary"
+         data-module="auto-track-event"
+         data-track-category="userAlerts:<%= alert_type %>"
+         data-track-action="postcodeErrorShown:<%= @location_error.postcode_error %>"
+         data-track-label="<%= t(@location_error.message, @location_error.message_args) %>">
+
```

Comment:
:+1: for this, moving to using the auto tracking is good and we get to burn an extra partial.  yay!